core/runtime/ops.rs

```diff
@@ -512,6 +613,64 @@ mod tests {
     }
   }
 
+  /// Run a test for a single op.
+  async fn run_async_test(
+    repeat: usize,
+    op: &str,
+    test: &str,
+  ) -> Result<(), AnyError> {
+    let mut runtime = JsRuntime::new(RuntimeOptions {
+      extensions: vec![testing::init_ops_and_esm()],
+      ..Default::default()
+    });
```

Comment:
Random thought: we do this a lot, maybe we should add `RuntimeOptions::with_extensions()` to remove some of these verbose lines?